.github/workflows/build.yml

```diff
@@ -0,0 +1,47 @@
+name: Build & Test
+on:
+  push:
+    branches:
+      - master
+      - release-*
+  pull_request:
+env:
+  DOTNET_NOLOGO: true
+jobs:
+  build:
+    name: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ windows-2019, ubuntu-18.04 ]
```

Comment:
Note that `ubuntu-20.04` is now the "current" version, and the `ubuntu-latest` label has been changed to point to that.

Reply:
GitVersion doesn't properly work yet with that version of Ubuntu

> So, there is a reason not to use latest ubuntu.... it breaks GitVersion
> We would need to move up to a newer GitVersion (which they completely changed to a new package) to get a version of LibGit2Sharp

also, there is

https://github.com/GitTools/GitVersion/issues/2642

So we went back to use this version since this is more or less a demonstration PR and not a "you must absolutely do it this way"

Reply:
With the newest version of Particular.Packaging and GitVersion we can now use that ubuntu version. See 📗 https://github.com/Particular/NServiceBus/runs/2288983832?check_suite_focus=true